microsoft-edge/toc.yml

```diff
@@ -848,6 +848,9 @@
             - name: View extensions analytics dashboard
               href: extensions-chromium/publish/extensions-analytics.md
 
+            - name: Respond to user reviews
+              href: extensions-chromium/publish/user-reviews.md
```

Comment:
Having more detailed URL's will help in Search engine optimization. Can we use something like reply-to-user-review-for-edge-extensions.md

Reply:
fyi
https://review.docs.microsoft.com/en-us/help/contribute/file-path-guidelines?branch=main#name-your-files
"Use action verbs that are specific, such as develop, buy, build, and troubleshoot. "
"Don't use "-ing" words."
Don't use unnecessary small words, such as a, or, and, the, and in."

I'm crafting this resulting now:
https://docs.microsoft.com/microsoft-**edge**/**extensions**-chromium/publish/**reply**-**user-reviews**
This path will fit with Docs Contributor Guide & this repo's conventions & has the requested key words, which are "reply user review edge extensions"